track_and_trace/README.md

```diff
@@ -0,0 +1,25 @@
+# Track and Trace
+
+The Grid Track and Trace transaction family allows users to track goods as they move through a supply chain. Records for goods include a history of ownership and custodianship, as well as histories for a variety of properties such as temperature and location. These properties are managed through a user-specifiable system of record types. This readme  outlines how to start an instance of Track and Trace locally.
```

Comment:
Delete the extra space after "This readme"

Also, README is usually written in all caps.